src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -367,6 +384,9 @@ ucs_status_t uct_rc_verbs_ep_flush(uct_ep_h tl_ep, unsigned flags,
         return UCS_OK;
     }
 
+    /* reset fence mode */
+    uct_rc_ep_fm_reset(&iface->super, &ep->fi);
```

Comment:
why reset here? flush is not completed yet. Would be valid to reset if nothing to be flushed